src/main/java/com/keniu/controllers/BookController.java

```diff
@@ -0,0 +1,56 @@
+package com.keniu.controllers;
```

Comment:
```suggestion
package com.keniu.controller;
```

Reply:
Why?

Reply:
Usually package name singular

Reply:
Usually package name plural)

Reply:
I disagree with you. Controller refers to a single concept, whereas controllers implies multiple classes, which is better for scalability. In Java, it's common to name packages in the plural form when they contain groups of similar classes: services, models, utils. Of course, if the project already follows a different naming convention, it’s better to stick to the existing one. However, when starting a new project, it’s a matter of preference, but generally, using the plural form is the accepted convention.

Reply:
Matter of personal preference/project conventions. From my experience, the singular form is more common